FEXCore/Source/Interface/IR/AOTIR.cpp

```diff
@@ -22,399 +22,383 @@
 
 
 namespace FEXCore::IR {
-  AOTIRInlineEntry *AOTIRInlineIndex::GetInlineEntry(uint64_t DataOffset) {
-    uintptr_t This = (uintptr_t)this;
+AOTIRInlineEntry *AOTIRInlineIndex::GetInlineEntry(uint64_t DataOffset) {
```

Comment:
Seems like the star/ampersand placement is very irregular. It's using `AOTIRInlineEntry *` here (space between type and star) but `auto& [Name, rv] = *reinterpret_cast<ArgsRV_t*>(ArgsRV);`/`HostToGuestTrampolinePtr* TrampolineAddress` in Thunks.cpp. Any idea why it's like that?

I can see why it might choose not to put a space when the type is used in isolation (but then it's not doing that below for the `(AOTIRInlineEntry *)` cast). Also `auto &[Name, rv]` would look kind of weird, though I could get used to it. But everything else should be more consistent.

Reply:
Setting `PointerAlignment` and `ReferenceAlignment` to `Left` aids with consistency, removed `DerivePointerAlignment` that was causing inconsistencies.